### DISCUSSION TITLE
Issue with converting .txt to .csv using R

### DISCUSSION THREAD

**Discussion by Divyanshi09** [jspsych/jsPsych#1601] [Q&A]
Hello,

I uploaded a jsPsych task on Jatos and I have been trying to convert the .txt result file to .csv and I am failing badly. I tried R with the instructions that you have given here: https://github.com/becky-gilbert/r-jatos-json-parser/blob/fef9618d9173a4033e2a1e200c0d13da530c02c9/parse_JSON_data_examples.R

I am certain that my task only has one component but the error that I am being shown is "Warning: file jatos_results_20210302201158 does not contain the expected number of components. Skipping this file.
Expected 1 components, found 24 JSON lines.
NULL"
I have attached the .txt file that I am trying to convert. Also, is there some other alternative to converting it to .csv or are programming platforms the only available options? I would be extremely grateful if you could help me out with this! 

[jatos_results_20210302201158.txt](https://github.com/jspsych/jsPsych/files/6071565/jatos_results_20210302201158.txt)

**Comment by becky-gilbert** [comment]
Hi @Divyanshi09, apologies for the very very slow reply :frowning: I thought I'd reply anyway, just in case it's still helpful to you and/or anyone else.

The R parsing function that I wrote is meant to parse JATOS result files that contain one line of JSON-formatted data per component, and save the data into CSV files. The text data you shared is already in CSV format. 

To open your text data in Excel, you can open a blank Excel worksheet, then go to Data -> From Text, and select your file. Then choose the "Delimited" option, and tick the box that says "My data has headers". On the next screen, choose "Comma" as the only delimiter. Then click "Next", "Finish", and "OK".
